Add SLE BCI verification with Podman (#101)
SUSE/bci-docs: content/guides/verify-sle-bci.adoc

`/etc/containers/registries.d/` with a filename of your choice.

=== Update signature verification policy (`/etc/containers/policy.json`)
Next, update the https://github.com/containers/image/blob/main/docs/containers-policy.json.5.md[/etc/containers/policy.json] file.
Under the `docker` attribute, add the `registry.suse.com` configuration similar to the following:

[source,json]
----
{
    "default": [
        {
            "type": "insecureAcceptAnything"
        }
    ],
    "transports":
        {
        "docker-daemon":
            {
                "": [{"type":"insecureAcceptAnything"}]
            },
	    "docker":
		{
		    "registry.suse.com": [{
			   "type": "sigstoreSigned",
			   "keyPath": "/usr/share/pki/containers/suse-container-key.pem",
			   "signedIdentity": {"type": "matchRepository"}
			}]
		}
	}
}
----
The specified configuration instructs Podman, Skopeo and Buildah to verify images under the
`registry.suse.com` repository. This way, before pulling the image, Podman checks the validity of the signature using the specified public key, and rejects the image if the validation fails.

{{< hint type="note" >}}

> If there are entries already existing under `transports.docker`, make sure you don't remove them and simply add another entry as explained in the preceding section.

Reviewer: I know it's technically not a list but a hash. However this is an end-user documentation and an end-user will probably get only confused if we talked about hashes.
```suggestion
Do not remove existing entries in `transports.docker-daemon.docker`. Instead append the entry for `registry.suse.com` to the list.
```

Reviewer: You are of course correct. I got fooled by the indentation…

Author: Yeah, sorry about that. Fixed.
